src/main.rs

```diff
@@ -402,6 +404,28 @@ pub fn setup_csrs() {
     trap::install_trap_handler();
 }
 
+#[cfg(target_feature = "v")]
+fn verify_vector_registers() {
+    // Because we just ran setup_csrs(), we know vectors are off
+    // Turn vectors on
+    CSR.sstatus.read_and_set_bits(0x200);
```

Comment:
Nit: Consider defining a const value for 0x200.

Reply:
There should be a way of getting from the tock register definition, but I couldn't figure it out (in 2 minutes) and I knew 0x200 off the top of my head. 

Will Fix